src/core.js

```diff
@@ -68,7 +68,7 @@ exports.register([
 // locales en and en-US are required for default behavior
 exports.register([
     require('./locale-en'),
-    require('./locale-en-US')
+    require('./locale-en-us')
```

Comment:
how did this not fail for me locally but only on CI?

Reply:
Added a check for case correctness of all the requires https://github.com/plotly/plotly.js/pull/2207/commits/43b73e0a6a54bdc0586973696d1f0c9cc2bf0f79